packages/xo-server/src/api/pool.mjs

```diff
@@ -116,6 +116,49 @@ setPoolMaster.resolve = {
 
 // -------------------------------------------------------------------
 
+export async function disableHa({ pool }) {
+  await this.hasPermissions(this.apiContext.user.id, [[pool, 'administrate']])
+
+  await this.getXapi(pool).disableHa()
+}
+
+disableHa.params = {
+  pool: {
+    type: 'string',
+  },
+}
+
+disableHa.resolve = {
+  pool: ['pool', 'pool', 'administrate'],
+}
+
+// -------------------------------------------------------------------
+
+export async function enableHa({ pool, heartbeatSrs, configuration }) {
+  await this.hasPermissions(this.apiContext.user.id, [[pool, 'administrate']])
+
+  await this.getXapi(pool).enableHa(heartbeatSrs, configuration)
+}
+
+enableHa.params = {
+  pool: {
+    type: 'string',
+  },
+  heartbeatSrs: {
+    type: 'array',
+    items: { type: 'string' },
+  },
```

Comment:
Should this take IDs instead of refs? /cc @julien-f

Reply:
xo-server API should use xo-server object ids and not opaque refs indeed.